### ISSUE TITLE
Wrong number of models in data available graph on PPTC provider page

### ISSUE THREAD

**Issue by zperova** [PDXFinder/pdxfinder#550]
**Describe the bug**
The number of models should be 261 according to the data in the database.

**To Reproduce**
1. Go to https://www.pdxfinder.org/source/pptc/
2. Scroll down to Data available
3. See error

**Expected behavior**
The number of models for each data type is 261

**Screenshots**
![Screenshot 2021-03-03 at 14.23.58.png](https://images.zenhubusercontent.com/5b631938e115f66327ad7ac6/494a5b1c-e354-41b4-bcb9-9656de2e2307)

**Comment by Afollet** [comment]
I just pushed these changes to dev. The count is correct on the here. However, every sample was linking to these three data type on the search. The sampleplatform sheet contained rows for all of the models when only a subset of those models have the WES, targeted_capture, and RNA-Seq data. I removed the ones not associated with these data types. Therefore, not all of the models in the search will now link to these datatypes.
